### ISSUE TITLE
Add ability to configure assignPublicIp of awsvpcConfiguration block

### ISSUE THREAD

**Issue by davivc** [azavea/django-ecsmanage#23]
I was getting this message when trying to run commands on my local machine:

``
Status reason | CannotPullContainerError: Error response from daemon: Get https://****.dkr.ecr.us-west-1.amazonaws.com/v2/: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
``

I did some digging and I've found this here https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_cannot_pull_image.html

"When a Fargate task is launched, its elastic network interface requires a route to the internet to pull container images. If you receive an error similar to the following when launching a task, it is because a route to the internet does not exist"

So I've added the parameter 'assignPublicIp' to network configuration and it worked properly:

```
network_configuration = {
    "awsvpcConfiguration": {
        "subnets": [subnet_id],
        "securityGroups": [security_group_id],
        "assignPublicIp": "ENABLED"
    },
}
```

Is there any option to add this to the config env vars?

**Comment by hectcastro** [comment]
Hey, @davivc, thanks for opening this issue.

There isn't a way to specify a value for `assignPublicIp` in the `django-ecsmanage` configuration, but this issue can be repurposed to add that as a feature. I think the reason why we haven't run into this ourselves is because we launch all ECS services inside of private subnets with NAT gateways attached. That allows us keep `assignPublicIp` disabled.

As an alternative, I believe that the equivalent of `assignPublicIp` can be set at the subset level. If you have the ability to control that, ECS may use that as your default vs. having to specify on every task that you need it on.

**Comment by davivc** [comment]
Yes, I think this approach is the best choice between the two. I'm fairly new on AWS Fargate/ECS and I'm migrating a project from GCP K8s and learning on the fly as well. 

I'll try to run the task with these NAT gateways attached to avoid this problem. 

Thas

**Comment by coalkettler** [comment]
@davivc We've implemented this feature in #28. You can now set `ASSIGN_PUBLIC_IP` to either `ENABLED` or `DISABLED` in your environment configuration dictionary. Thanks for opening the issue and for your patience!
